samples/client/petstore/c/build-and-test.bash

```diff
@@ -22,4 +22,4 @@ cmake .
 
 make 
 
-ctest
+ctest -j <threads>
```

Comment:
Sorry if I was unclear: With <threads> I meant that you need to specifiy the amount of threads you want to use. I don't know how many cores your CI has, that is why I wrote it like that.
e.g.: $ ctest -j 4

Reply:
Likely just one core so we should just go with `ctest` without the -j argument.